src/services/localSSHService.ts

```diff
@@ -75,6 +79,7 @@ export class LocalSSHService extends Disposable implements ILocalSSHService {
     async extensionServerReady(): Promise<boolean> {
         try {
             await canExtensionServiceServerWork();
+            this.metricsReporter.reportPingExtensionStatus(this.hostService.gitpodHost, 'success');
```

Comment:
should not we pull out flow at the begging of the method and then use `flow.gitpodHost` and below as well